### PR TITLE
Extract memLogPeak() into function

### PR DESCRIPTION
**Pull request by richardleggett**
## Description
Extract logging of peak CHIP/RAM usage into callable function in memory.c/h for debugging purposes.

## Checklist
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by richardleggett** [comment]
I'm not 100% sure about the codestyle or ordering so please let me know on those and I'll keep in mind for future.

File: src/ace/managers/memory.c [hunk 1]
```diff
@@ -190,10 +190,7 @@ void *_memAllocDbg(
 			"[MEM] ERR: couldn't allocate %lu bytes! (%s:%u)\n",
 			ulSize, szFile, uwLine
 		);
-		logWrite(
-			"[MEM] Peak usage: CHIP: %lu, FAST: %lu\n",
-			s_ulChipPeakUsage, s_ulFastPeakUsage
-		);
+		memLogPeak();
```
Comment (richardleggett) [inline src/ace/managers/memory.c:193]: Should this also stay inlined to avoid a function call?

Comment (tehKaiN) [reply src/ace/managers/memory.c:193]: it's only for debug builds, so I think a fn call is okay - it won't be fast anyway ;)

**Review by tehKaiN — APPROVED** [review]

File: src/ace/managers/memory.c (hunk 1 above)
Comment (richardleggett) [reply src/ace/managers/memory.c:193]: That's what I hoped, good to hear. Did I see that CI failed on this, false alarm?

**Pull request merged**